stub/stub.go

```diff
@@ -60,6 +59,7 @@ type Input struct {
 }
 
 type Output struct {
+	Delay Duration          `json:"delay"`
```

Comment:
I think we can't do it directly under Output struct, can you please create a Meta field and put duration under it?

Reply:
@shubhjha123 I mean like this
```
type Output struct{
  Meta Meta
}

type Meta struct{
  Delay Duration
}
```